techmatters/flex-plugins
# home page shortcut link to counselor toolkit

Primary reviewer: @stephenhand 

## Description

- New feature: Home page shortcut link to counselor toolkit

### Checklist
- [N/A] New tests added
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes CHI-1547

### Verification steps

- Verify that there is an information icon on the side nav
- Click on the information Icon on the side nav
<img width="189" alt="Screenshot 2023-02-02 at 11 20 05 AM" src="https://user-images.githubusercontent.com/40429848/216298462-1c7cee79-6bca-46ef-9e3c-926855098e9f.png">
